src/plays/simple-live-chat/components/chat/chat-body.jsx

```diff
@@ -0,0 +1,132 @@
+import format from "date-fns/format";
+import { Fragment, useEffect, useRef, useState } from "react";
+
+// css
+import "./chat-body.scss";
+
+// assets
+import userIcon from "../../images/user_icon.png";
+
+const User = ({ photoURL, onClick }) => {
+  const [defaultImage, setDefaultImage] = useState(photoURL);
+  return (
+    <img
+      src={defaultImage}
+      alt='avatar'
+      onClick={onClick}
+      className='simple-live-chat-user'
+      onError={() => setDefaultImage(userIcon)}
+    />
+  );
+};
+
+const ChatBody = ({ message, uid, loading }) => {
+  const [node, setNode] = useState([]);
+  const scrollHandler = useRef();
+
+  const dateTimeFormatter = (val) => {
+    return format(new Date(val * 1000), "hh:mm bbb");
```

Comment:
can you declare all constants at the top or in a separate file. e.g: date format.

Reply:
this is fixed